cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
The `validate` method in TicketSerializer is calling a static method `validate_ticket` from the Ticket model. This is not a good practice as it couples the model layer with the serializer layer. Consider moving this validation logic directly into the serializer or use Django's form validation.